cookbook/core/type_system/structured_dataset.py

```diff
@@ -87,18 +87,69 @@ def get_subset_df(
 ) -> Annotated[StructuredDataset, subset_cols]:
     df = df.open(pd.DataFrame).all()
     df = pd.concat([df, pd.DataFrame([[30]], columns=["Age"])])
-    # On specifying BigQuery uri for StructuredDataset, Flytekit will write pd.dataframe to a BigQuery table
+    # On specifying BigQuery uri for StructuredDataset, flytekit writes pd.dataframe to a BigQuery table
     return StructuredDataset(dataframe=df)
 
+# %%
+# Example of StructuredDataset With ``uri`` Argument
+# ==================================================
+#
+# As mentioned previously, if you specify BigQuery ``uri`` for StructuredDataset, flytekit writes the pandas dataframe to a BigQuery table.
+#
+# To write a dataframe to a BigQuery table, 
+#
+# #. Use your gcp account (or `create <https://cloud.google.com/docs/authentication/getting-started>`_ one) and create a service account.
+# #. Create a project and add the ``GOOGLE_APPLICATION_CREDENTIALS`` environment variable to your bashrc file.
+# #. Create a dataset in your project.
+#
+# Let's dive into the example. 
+
+# %%
+# Import the dependencies.
+import pandas as pd
+from flytekit import task
+from flytekit.types.structured import StructuredDataset
+
+# %%
+#.. note:: The BigQuery uri's format is ``bq://<project_name>.<dataset_name>.<table_name>``.
 
 # %%
+# Define a task that converts a pandas dataframe to a BigQuery table.
+@task
+def pandas_dataframe_to_bq_table() -> StructuredDataset:
+    # create a pandas dataframe
+    df = pd.DataFrame({"Name": ["Tom", "Joseph"], "Age": [20, 22]})
+    # convert the dataframe to StructuredDataset
+    return StructuredDataset(dataframe=df, uri="bq://sample-project-1-352610.sample_352610.test1")
+
+# %%
+# Define a task that converts a BigQuery table to a pandas dataframe .
+@task
+def bq_table_to_dataframe(sd: StructuredDataset) -> pd.DataFrame:
+    # convert to pandas dataframe
+    return sd.open(pd.DataFrame).all()
+
+# %%
+#.. note:: Flyte creates the table inside the dataset in the project upon BigQuery query execution.
```

Comment:
Should this note be present after the pandas_dataframe_to_bq_table task? Also, can we rename the tasks to `pandas_to_bq` and `bq_to_pandas`?

Reply:
I thought so too, but after your review (you have asked me to change the place of the first note), so two notes in the same section may not look too good.